.gitignore

```diff
@@ -25,6 +25,7 @@ public/client
 public/system
 public/uploads
 
+disabled_plugins/
```

Comment:
this is just temporary I think. Just a place I can move a plugin so it's out of the way